pkg/connector/inet/inet.go

```diff
@@ -141,7 +143,7 @@ type Connection struct {
 // NewConnection creates a Connection.
 func NewConnection(vin string, authHeader, serverURL, userAgent string) *Connection {
 	conn := Connection{
-		UserAgent:  userAgent,
+		UserAgent:  userAgent + ProxyProtocolVersion,
```

Comment:
Have `pkg/proxy/proxy.go` modify the UserAgent field if its Connections. This will allow the backend to distinguish clients using the proxy package from native Go clients that use the inet Connector directly.